### ISSUE TITLE
Improve design

### ISSUE THREAD

**Issue by simoncozens** [simoncozens/fontbakery-ui#2]
Currently looks horrible

**Comment by davelab6** [comment]
Screenshot pls

**Comment by eliheuer** [comment]
On macOS Big Sur 11.0.1:
<img width="1062" alt="fbgui" src="https://user-images.githubusercontent.com/5162664/99481317-b4879f00-290e-11eb-8965-cc8e72aa7da2.png">

**Comment by felipesanches** [comment]
yeah... the top of the report looks reasonable, but then the rest is super weird:

<img width="691" alt="Captura de Tela 2021-03-15 às 17 01 40" src="https://user-images.githubusercontent.com/213676/111214930-4c418500-85b1-11eb-8348-d76b0129b811.png">

And no need to print the full file path many-many-many times!
<img width="628" alt="Captura de Tela 2021-03-15 às 17 01 26" src="https://user-images.githubusercontent.com/213676/111214983-5b283780-85b1-11eb-8c3b-84db3c4a8f30.png">


To be fair, though, I think the bad rendering is in part related to the different level of support for Markdown+HTML5 syntax between this Qt application and the GitHub issue tracker / PR UI.

**Comment by felipesanches** [comment]
I think that we can change the markdown generated by FontBakery Core to print the fullpath to the fonts directory on the top of the report and then only print relative file paths in the rest of the report.

We probably never saw this bad layout of the report because we were mostly running fontbakery from the command line to check fonts in the local directory (at least most of the times). And now the FB-UI deals with full paths because that's probably what the drag-n-drop API gives us.

**Comment by felipesanches** [comment]
the blue highlight when I click on a collapsed box is also pretty ugly:
<img width="631" alt="Captura de Tela 2021-03-15 às 17 18 56" src="https://user-images.githubusercontent.com/213676/111216020-a2fb8e80-85b2-11eb-9644-16476d4433b2.png">
